src/app/shared/store/provider.state.ts

```diff
@@ -507,7 +507,7 @@ export class ProviderState {
     { dispatch }: StateContext<ProviderStateModel>,
     { payload }: BlockProviderAdminById
   ): Observable<void | Observable<void>> {
-    return this.providerAdminService.blockProviderAdmin(payload.userId, payload.providerId).pipe(
+    return this.providerAdminService.blockProviderAdmin(payload.userId, payload.providerId, payload.isBlocked).pipe(
```

Comment:
a great amount of params - a sign of smell code, so consider to create an object